arm-sql/2014-04-01/swagger/connectionPolicies.json

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
Please fix the linter  error for this file.  Error is: Operations API must be implemented for '/providers/Microsoft.Sql/operations'.